app/locales/en.json

```diff
@@ -80,34 +82,35 @@
     "maps_import_disclaimer": "Safe Paths has no affiliation with Google and never shares your data.",
     "maps_import_text": "To see if you encountered someone with COVID-19 prior to downloading this app, you can import your personal location history.",
     "maps_import_title": "Google Maps",
+    "nCoV2019_url_info": "For more information on the dataset for this map",
     "news_subtitle": "Read about the latest COVID updates from your health authority and in general.",
     "news_title": "Latest news",
     "no_data": "No data",
+    "notification_2_weeks_ago": "2 weeks ago",
+    "notification_data_not_available": "No exposure data is available.",
+    "notification_select_authority": "Select Healthcare Authority",
+    "notification_title": "2-Week Exposure Profile",
+    "notification_today": "today",
+    "notification_warning_text": "If a Healthcare Authority exists in your area you can subscribe to obtain regular updates of exposure risks.",
+    "notifications_exposure_format": "%1$s days ago you crossed paths someone infectious for %1$s minutes.",
+    "notifications_exposure_format_today": "Today you crossed paths with someone infectious for %1$s minutes.",
+    "notifications_exposure_format_yesterday": "Yesterday you crossed paths with someone infectious for %1$s minutes.",
+    "notifications_no_exposure": "No known exposures to COVID-19 during the last two weeks.",
+    "overlap_found_button_label": "Public Data Loaded",
+    "overlap_no_results_button_label": "Public Data Loaded",
+    "overlap_para": "{\"one\":\"The green trail represents your location history\\n\\nThe light purple circles represent the public dataset\",\"other\":\"\"}",
```

Comment:
this is incorrect. needs fixing in lokalise

Reply:
do we need to fix this in both places or just on lokalise?

Reply:
We need to fix in both places, I will make a PR now

Reply:
Fixing in #648